students/PadmajaPeri/Lesson05/Lesson05_logging_exercise.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import time
+from logging.handlers import SysLogHandler
+
+# Constant that represents the port that the server listens on. This server
+# will get the logs
+SYSLOG_UDP_PORT = 514
+
+# Formatter for logs written to stream and file
+format = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
```

Comment:
`format` is a built-in method.  You'll want to choose a different variable name.